### DISCUSSION TITLE
【委托】急急急急急

### DISCUSSION THREAD

**Discussion by dveam** [share121/inter-knot#158] [其他]
![IMG_20240716_054957](https://github.com/user-attachments/assets/374cd1be-0c39-4a55-92af-046844accfda)
先表明一下我的身份——用你们的话来说，我是叛军。所以绳网上的各位不必害怕。最近的《城市快报》大家都看了吧！
![IMG_20240809_122828](https://github.com/user-attachments/assets/8c1977d8-26d9-4cf1-985c-e4fdc4daa0af)
TT本来就打不过防卫军，斗智也斗不过这些防卫军。现在还要升级装备，真是太可恶啦！作为卧底在驻地的叛军，我有一些情报：防卫军在七月找人升级了一些载具以对抗叛军，但当时由于我们叛军的捣乱，并未完成交付，所以这批载具最近才要运来。
![IMG_20240809_200530](https://github.com/user-attachments/assets/b4a527e0-9688-42f7-b255-7fc969dd45b7)
我需要一些在空洞履历丰富的人，配合我们劫持这批载具。而且防卫军也算是绳匠的敌人吧？所以说，这是双赢。
委托费：丁尼x1000 菲林x1 没有了

**Comment by wushanef** [comment]
这个委托我接了

**Reply by dveam** [reply]
您就是大善人！

**Comment by DeepslateQAQ** [comment]
就1个菲林？打发叫花子呢.jpg
<strong><span style="color: #ff0000">不接</span></strong>

**Reply by dveam** [reply]
我只有这么多了😭

**Comment by nahida0-o** [comment]
或许你可以尝试联系一下我们须弥的三十人团？不过在虚空终端被我禁止使用之后到还没再广泛推行什么可以上网的用具，可能需要你亲自来一趟须弥了。不过委托费确实太少了，大部分绳匠应该都不会愿意接，如果钱多的话我相信他们就算不会也会去学习以完成你的委托。[动画表情]

**Reply by dveam** [reply]
这个“须弥”在哪里呀？我没找到TT   这些委托费是我所能拿出的最多了😭

**Comment by ink808** [comment]
1000丁尼一菲林换我们去跟防卫军打，我打防卫军真的假的？(^_^)

**Reply by nahida0-o** [reply]
会赢吗

**Reply by dveam** [reply]
钱财乃是身外之物

**Comment by XiaoFffeng** [comment]
这么点报酬，防卫军都不敢这么干，你们叛军想干出点名堂总得舍得出钱吧

**Reply by dveam** [reply]
我们被防卫军打的只剩这些了TT对防卫军作战时防卫军和我们的损失之比是1:5😭组织还收了我们好多军费上去

**Comment by qwq1949** [comment]
![ab363859cf021d992c2c8beccc4837595f409bc0](https://github.com/user-attachments/assets/aba142eb-bae6-485d-bb0e-ddd922e7b9ee)
我~~淫~~河球棒侠是你一个菲林就能打发的？

**Reply by dveam** [reply]
“银河球棒侠”？😃好像在哪听说过，一定是一位优秀可靠的绳匠吧。这位大侠！钱财乃是身外之物！考虑考虑帮帮我们吧！😭

**Reply by qwq1949** [reply]
这位匠友看起来挺着急的啊
那好吧 我银河球棒侠勉强帮帮你吧
其实我不是很厉害 但是我认识一个好厉害的人 🤓
叫 花火 ！你叫她来 包您满意！🤓

**Reply by dveam** [reply]
这位大侠！感激之情无以言表呐！可否给我一个您这位叫“花火”的友人的联系方式？多谢多谢！😃

**Reply by qwq1949** [reply]
那当然是 10086 啦🤓

**Comment by ZhuYuan-148** [comment]
治安官已出动。
![Screenshot_20240810_180118](https://github.com/user-attachments/assets/e655c395-87b5-4dde-be9a-15efa0337dc9)

**Reply by dveam** [reply]
😱😱😱😱😱

**Reply by dveam** [reply]
她真好看😍不是，我是说😱治安官？！😱

**Comment by sky27950** [comment]
30个菲林我考虑考虑

**Reply by dveam** [reply]
分期付款可以么？

**Comment by Lanxey** [comment]
你明明可以白嫖，却还是给了1菲林，我哭死

**Reply by dveam** [reply]
TT只剩下这些力

**Comment by aaa00073** [comment]
1菲林？我买个蓝色音擎都要160菲林！

**Reply by dveam** [reply]
这样你不是有1/160个蓝色音擎了么，只需要再接159次这样的委托就可以买蓝色音擎了😃

**Comment by w1-w2-w3** [comment]
一个苹果，一瓶蒲公英酒，这个委托我接了

**Reply by dveam** [reply]
真的只需要这些么？真是太感谢了！世界上还是好人多啊！

**Comment by Larksonguesinaspic** [comment]
不用求助了，我已经打电话和13号讲了
等着被大记忆恢复术罢(悲

**Reply by dveam** [reply]
泥。。TT
